src/handlers/user/loginHandler.ts

```diff
@@ -1,6 +1,22 @@
-const loginHandler = (
+const {User} = require('../../db')
+const bcrypt = require('bcrypt')
+
+
+const loginHandler = async (
   email: string | undefined,
   password: string | undefined
-) => {};
+) => {
+  if(!email || !password) throw new Error ('Missing data.')
+
+  const user = await User.findOne({where: email})
+
+  if(!user) throw new Error ("User doesn't exists...")
```

Comment:
## Spelling mistake
There's a little spelling mistake here:
Before:
```javascript
if(!user) throw new Error ("User doesn't exists...")
```
After:
```javascript
if(!user) throw new Error ("User doesn't exist.") // exist instead of exists
```